datajoint/settings.py

```diff
@@ -137,7 +137,8 @@ def get_store_spec(self, store):
         spec['subfolding'] = spec.get('subfolding', DEFAULT_SUBFOLDING)
         spec_keys = {  # REQUIRED in uppercase and allowed in lowercase
             'file': ('PROTOCOL', 'LOCATION', 'subfolding', 'stage'),
-            's3': ('PROTOCOL', 'ENDPOINT', 'BUCKET', 'ACCESS_KEY', 'SECRET_KEY', 'LOCATION', 'secure', 'subfolding', 'stage')}
+            's3': ('PROTOCOL', 'ENDPOINT', 'BUCKET', 'ACCESS_KEY', 'SECRET_KEY', 'LOCATION',
+                    'secure', 'subfolding', 'stage', 'proxy_server')}
```

Comment:
Whoops... Looks like the tests failed because of a small indention error (related to PEP8 styling). To fix it, we just need to remove a space like so:

```suggestion
            's3': ('PROTOCOL', 'ENDPOINT', 'BUCKET', 'ACCESS_KEY', 'SECRET_KEY', 'LOCATION',
                   'secure', 'subfolding', 'stage', 'proxy_server')}
```

Reply:
Indeed, I haven't seen that before as well 😅